b2share/modules/schemas/alembic/35d7d8958395_create_schemas_tables.py

```diff
@@ -0,0 +1,106 @@
+"""Create schemas tables.
+
+Revision ID: 35d7d8958395
+Revises: 226892354af5
+Create Date: 2017-04-25 18:03:15.919582
+
+"""
+from alembic import op
+from datetime import datetime
+import sqlalchemy as sa
+import uuid
+
+from sqlalchemy.types import TIMESTAMP
+from sqlalchemy_utils.types import UUIDType
+from sqlalchemy.engine.reflection import Inspector
+
+
+# revision identifiers, used by Alembic.
+revision = '35d7d8958395'
+down_revision = '226892354af5'
+branch_labels = ()
+depends_on = 'fb99eeaec4ac'  # in b2share communities
+
+
+def upgrade():
+    """Upgrade Database."""
+    op.create_table(
+        'b2share_root_schema_version',
+        sa.Column('version', sa.Integer, autoincrement=False),
+        sa.Column('json_schema', sa.Text, nullable=False),
+        sa.PrimaryKeyConstraint('version')
+    )
+    op.create_table(
+        'b2share_block_schema',
+        sa.Column('created', TIMESTAMP, nullable=False),
+        sa.Column('updated', TIMESTAMP, nullable=False),
+        sa.Column('id', UUIDType, default=uuid.uuid4),
+        sa.Column('name', sa.String(200),
+                  sa.CheckConstraint('LENGTH(name) > 2 AND '
+                                     'LENGTH(name) <= 200',
+                                     name='ck_b2share_block_schema_name'),
+                  nullable=False),
+        sa.Column('deprecated', sa.Boolean, default=False, nullable=False),
+        sa.Column('community', UUIDType, nullable=False),
+        sa.PrimaryKeyConstraint('id', name='pk_b2share_block_schema'),
+        sa.ForeignKeyConstraint(['community'], [u'b2share_community.id'],
+                                name='fk_b2share_community_id')
```

Comment:
Are you sure that this is the name of the constraint when B2SHARE v2.0.0 was released? the name convention `fk_*` only exists on invenio-db master branch.

This concerns also the other constraint names.